### ISSUE TITLE
server_default poorly documented

### ISSUE THREAD

**Issue by sqlalchemy-bot** [sqlalchemy/sqlalchemy#3086]
**Migrated issue, originally created by Michael Bayer ([@zzzeek](https://github.com/zzzeek))**

it only includes text().  illustrate how enough information to support an example like:

```
tbl = Table("derp", metadata,
    Column("arr", ARRAY(Text),
                server_default=array([literal_column("'foo'"),
                                    literal_column("'bar'"),
                                    literal_column("'baz'")])),
)

```

that is, expressions, literals, etc.   as a task elsewhere we have to figure out if we should drop the literal_column aspect of this.

**Comment by jvanasco** [comment]
ARRAY/array is postgresql specific. i think this should be sqlite compatible.

anyone have an idea for a more applicable example than what @zzzeek provided?

I have an idea on how to structure this section, I'm just lost on what the example should be.

**Comment by sqla-tester** [comment]
**jonathan vanasco** has proposed a fix for this issue in the **master** branch:

**Fixes: #3086** https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3097

**Comment by jvanasco** [comment]
handled with the postgres array construct.

removed literal, as it was irrelevant to this example.
